unified-codes/apps/data-service/src/app/server.js

```diff
@@ -4,7 +4,7 @@ const start = async () => {
   let fastifyServer;
   try {
     fastifyServer = createFastifyServer(createApolloServer());
-    await fastifyServer.listen(4000);
+    await fastifyServer.listen(4000, '0.0.0.0');
```

Comment:
I feel like I should probably explain this - not having this specified means fastify only listens at 'localhost' or '127.0.0.1' (https://www.fastify.io/docs/latest/Server/) - however when running inside the container requests originate from the docker container's IP address, not localhost

Reply:
- Hostname - I was thinking the same thing @wlthomson and tried exactly what you mentioned (with 127.0.0.1 explicitly defined) - with this, the request passes through from localhost:4000 -> container but given the container is located inside a docker network, the container itself sees the connection as coming from the docker network rather than 127.0.0.1. I'm sure this behaviour can be customised with a custom network, but I didn't quite want to touch that yet 🙀 
- Port - Good idea to have in `.env` will update